lib/plain/plain-client.ts

```diff
@@ -20,6 +20,13 @@ export const createPlainClient = (
           action: 'get',
           params: { url, config },
         }),
+      patch: (url, payload, config) =>
```

Comment:
not sure why codecov is complaining, this is implicitly tested in `test/unit/entities/entry-test.js`